src/app/chart/sparkline/examples/sparkline-example.module.ts

```diff
@@ -8,16 +8,16 @@ import { SparklineExampleComponent } from './sparkline-example.component';
 import { ChartModule } from '../../chart.module';
 
 @NgModule({
-  declarations: [ SparklineExampleComponent ],
+  declarations: [SparklineExampleComponent],
   imports: [
     CommonModule,
     FormsModule,
     DemoComponentsModule,
     ChartModule,
     TabsModule.forRoot()
   ],
-  providers: [ TabsetConfig ]
+  providers: [TabsetConfig]
 })
 export class SparklineExampleModule {
-  constructor() {}
+  constructor() { }
```

Comment:
Which rule states there must be a space here?

Reply:
Not sure - will look to find it...